plugins/binanceExchange_ws_test.go

```diff
@@ -89,3 +103,70 @@ func Test_binanceExchangeWs_GetOrderBook(t *testing.T) {
 		assert.True(t, ob.Bids()[0].Volume.AsFloat() > 0)
 	}
 }
+
+func Test_binanceExchangeWs_GetLatestTradeCursor(t *testing.T) {
+
+	if testing.Short() {
+		return
+	}
+
+	startIntervalSecs := time.Now().Unix()
+
+	testBinanceExchangeWs, err := makeBinanceWs(emptyAPIKeyBinance)
+
+	if !assert.NoError(t, err) {
+		return
+	}
+
+	cursor, e := testBinanceExchangeWs.GetLatestTradeCursor()
+	if !assert.NoError(t, e) {
+		return
+	}
+	endIntervalSecs := time.Now().Unix()
+
+	if !assert.IsType(t, "string", cursor) {
+		return
+	}
+
+	cursorString := cursor.(string)
+	cursorInt, e := strconv.ParseInt(cursorString, 10, 64)
+	if !assert.NoError(t, e) {
+		return
+	}
+
+	if !assert.True(t, startIntervalSecs <= cursorInt, fmt.Sprintf("returned cursor (%d) should be gte the start time of the function call in millis (%d)", cursorInt, startIntervalSecs)) {
+		return
+	}
+	if !assert.True(t, endIntervalSecs >= cursorInt, fmt.Sprintf("returned cursor (%d) should be lte the end time of the function call in millis (%d)", cursorInt, endIntervalSecs)) {
+		return
+	}
+}
+
+func Test_binanceExchangeWs_GetTradeHistory(t *testing.T) {
+
+	if testing.Short() {
```

Comment:
makes sense to put this here because we can't run this in the CI pipeline since it needs API keys

were you able to test this with your own binance API keys?